pyveg/scripts/create_analysis_report.py

```diff
@@ -6,7 +6,7 @@
 import pypandoc
 
 
-def create_markdown_pdf_report(path, collection_name):
+def create_markdown_pdf_report(path, collection_name, do_timeseries, size_ts):
```

Comment:
rather than passing "size_ts" here, and then in the function doing "if size_ts > 12",  maybe have in this file
```
do_ews = ( size_ts > MIN_NUM_POINTS_FOR_EWS)
```
and pass "do_ews" as the argument instead of "size_ts" ?